Updated learning resources and added an Info component (#2968)
nodejs/nodejs.dev: content/learn/manipulating-files/node-folders.en.md

});
```

> Removing a folder that has content can be more complicated than you need. Use `fs.rm()` and pass the option `{ recursive: true }` to recursively remove the contents.

Reviewer: ```suggestion
To remove a folder that has contents use `fs.rm()` with the option `{ recursive: true }` to recursively remove the contents.
```

});
```

> <Alert> <b>NOTE:</b> In Node <code>v16.x</code> the option <code>recursive</code> is <b>deprecated</b> for <code>fs.mdir</code> of callback API, instead use <code>fs.rm</code> to delete folders that have content in them</Alert>

Reviewer: Let's update the example to use the new way then instead of adding an alert?

As Node.js v16 is already "Maintenance LTS", meaning, people should already be using this new way.

Author: I updated the example as such:
![image](https://user-images.githubusercontent.com/84540554/199273269-6cae4cbb-a922-4ade-a185-660c2a766971.png)
